shell/common/shell_test_platform_view_metal.mm

```diff
@@ -4,52 +4,28 @@
 
 #include "flutter/shell/common/shell_test_platform_view_metal.h"
 
-#import <Metal/Metal.h>
-
 #include <utility>
 
+#include "flutter/shell/common/shell_test_platform_view_metal.h"
 #include "flutter/shell/gpu/gpu_surface_metal_impeller.h"
 #include "flutter/shell/gpu/gpu_surface_metal_skia.h"
 #include "flutter/shell/platform/darwin/graphics/FlutterDarwinContextMetalImpeller.h"
 #include "flutter/shell/platform/darwin/graphics/FlutterDarwinContextMetalSkia.h"
 
 FLUTTER_ASSERT_ARC
 
-namespace flutter {
-namespace testing {
+namespace flutter::testing {
 
-// This is out of the header so that shell_test_platform_view_metal.h can be included in
-// non-Objective-C TUs.
-struct DarwinContextMetal {
-  FlutterDarwinContextMetalSkia* skia_context;
-  FlutterDarwinContextMetalImpeller* impeller_context;
-  id<MTLTexture> offscreen_texture;
-};
```

Comment:
No longer necessary -- these fields are now directly on `ShellTestPlatformViewMetal` in the header.